bricks/dart_frog_dev_server/hooks/pre_gen.dart

```diff
@@ -72,11 +72,8 @@ Future<void> preGen(
 
   context.vars = {
     'port': context.vars['port'] ?? '8080',
-    'directories': configuration.directories
-        .map((c) => c.toJson())
-        .toList()
-        .reversed
```

Comment:
@felangel do you remember why this `reversed` call was in here?

If I kept it, it would "nullify" the ordering set to make that non dynamic routes takes precedence.

I tested without the `reversed` call and it seems to be working fine, but I believe that this call must have had a reason to be in here before, maybe you remember and could advise me which cases it covers so I can test with the new changes?

Reply:
iirc the idea was to order the directories from most nested to least nested (most specific to least specific) so that the router generation would mirror that order and routes like:

`/users/[id]/pets`

Would be registered and handled before routes like:

`/users/[id]`